src/dictionary.c

```diff
@@ -198,6 +198,15 @@ static void SpellCheckDictAuxSave(RedisModuleIO *rdb, int when) {
   dictReleaseIterator(iter);
 }
 
+static void SpellCheckDictAuxSave2(RedisModuleIO *rdb, int when) {
+  if (when == REDISMODULE_AUX_BEFORE_RDB) {
+    return;
+  }
+  if (dictSize(spellCheckDicts)) {
+    SpellCheckDictAuxSave(rdb, when);
+  }
```

Comment:
@nafraf What about adding 2 tests with `FT.DICTADD` and `FT.DICTDEL`  similar to the tests added with `FT.CREATE` and `FT.DROPINDEX`?

Reply:
@oshadmi  Thanks. Tests added.
`SpellCheckDictAuxSave2()` was modified:  if at least one dictionary has terms, all the dictionaries are saved.